spec/latest/index.bs

```diff
@@ -238,6 +238,30 @@ after reading the values array into memory, it should be interpreted as boolean
 For a given [[#key_format]], all named binary arrays for that format shall have a
 corresponding name in `data_types`.
 
+The following strings shall be used to describe data types:
+: "uint8"
+:: unsigned 8-bit integer
+: "uint16"
+:: unsigned 16-bit integer
+: "uint32"
+:: unsigned 32-bit integer
+: "uint64"
```

Comment:
Allow `iso[value]`, since we are using this square bracket model anyway.

Reply:
(This is not part of this PR, but just placing this comment here)

Reply:
I merged the iso PR into this one, since this one is introducing complex it was easier to just treat all of the value modifiers at once